app/views/episodes/_tabs.html.erb

```diff
@@ -4,6 +4,7 @@
       <%= tab_link_to @episode.title, edit_episode_path(@episode), class: "text-truncate", active: episode_metadata_active? %>
       <%= render "episode_media/tab", episode: @episode %>
       <%= tab_link_to t(".player"), episode_player_path(@episode) %>
+      <%= render "episode_transcripts/tab", episode: @episode %>
```

Comment:
Can you move the transcripts tab ahead of the Embeddable Player but after Media files (3rd in order)?